### ISSUE TITLE
Add method to return preconfigured client from MockServerRule

### ISSUE THREAD

**Issue by rajneeshpatel** [jamesdbloom/mockserver#188]
First off, this is a great project and already replacing my use of wiremock with this.

Was thinking given the typical use case/pattern below for using the JUnit rule and MockServerClient

``` java
@Rule
public MockServerRule mockServerRule = new MockServerRule(this);

@Test
public void shouldDoSomething() {
    new MockServerClient("localhost", mockServerRule.getHttpPort())
        .when(request()...
}
```

Instead of having to new up MockServerClient and configure with host and port, it would be convenient to have a method on the MockServerRule like getClient() so you can instead do the following

``` java
@Test
public void shouldDoSomething() {
    mockServerRule.getClient()
        .when(request()...
}
```

Or even

``` java

@Test
public void shouldDoSomething() {
    mockServerRule.client()
        .when(request()...
}
```

The client returned by .getClient()/client() would be pre-configured to "localhost" and the port the Mock Server is running on so by adding this method, saves some code and config.

Any mileage/appetite for this?

Thanks

Raj

**Comment by jamesdbloom** [comment]
This sounds like a good idea.  Currently however the test should not have to new up a client, instead the test has to have a field (`public` or `private`) of type `MockServerClient`.  As follows:

```java
public class MockServerRuleTest {

    @Rule
    public MockServerRule mockServerRule = new MockServerRule(this);

    private MockServerClient mockServerClient;

    @Test
    public void someTest() {
        ...
    }

}
```

I'm not sure why it was exactly done this way as I agree your suggestion sounds simpler.  However, I'm happy to play with you approach and extend the rule to support it.

**Comment by rajneeshpatel** [comment]
Sounds good. If its OK with you, will fork, have a play around and send you a pull request when ready. Will of course add tests around this in the request.

**Comment by jamesdbloom** [comment]
This is now implemented, this will be available in 5.2.4.

**Comment by rajneeshpatel** [comment]
Fantastic and thanks for this, looking forward to using.
